docs/general/ad-filtering/create-own-filters.md

```diff
@@ -536,7 +536,7 @@ If there is a `$~third-party` modifier, the rule is only applied to the requests
 
 :::info Compatibility
 
-Rules with the `$to` modifier are supported by AdGuard for Windows, Mac, and Android, **running CoreLibs version 1.12 or later**.
+Rules with the `$to` modifier are supported by AdGuard for Windows, Mac, and Android, **running CoreLibs version 1.12 or later**, and AdGuard Browser extension for Chrome, Firefox, Edge, **running TSUrlFilter v2.1.3 or later**.
```

Comment:
Is Opera intentionally omitted?

Reply:
i got that from $domain modifier compatibility string. Opera is also chromium-based

Reply:
Chromium and Firefox?

Reply:
maybe no need to specify browsers since all of them are supported? I think `AdGuard Browser extension **running ...` is enough

Reply:
```suggestion
Rules with the `$to` modifier are supported by AdGuard for Windows, Mac, and Android, **running CoreLibs v1.12 or later**, and AdGuard Browser Extension, **running TSUrlFilter v2.1.3 or later**.
```